pallets/settlement-groth16/src/benchmarking.rs

```diff
@@ -0,0 +1,38 @@
+use super::*;
+
+use crate::groth16::{Curve, Groth16};
+#[allow(unused)]
+use crate::Pallet as SettlementGroth16Pallet;
+use frame_benchmarking::v2::*;
+use frame_system::RawOrigin;
+
+#[benchmarks]
+mod benchmarks {
+    use super::*;
+
+    #[benchmark]
+    fn submit_proof_bn254(n: Linear<0, 16>) {
```

Comment:
Maybe here you can access to `<T as Config>::MAX_NUM_INPUTS`

Reply:
Fixed in https://github.com/HorizenLabs/zkVerify/pull/66/commits/222bb7568e108e7fe40ebbb95fcbdd835926f01d